src/shared/components/layouts/styles/sidebar-layout.module.scss

```diff
@@ -35,6 +35,10 @@
     &::-webkit-scrollbar {
       display: none;
     }
+    // Hides the facets as they appear until the page content loads
+    @media only screen and (max-width: 640px) {
+      display: none;
```

Comment:
It's good to use display none in CSS because it means the browser should avoid doing the work of rendering the content, but we could also avoid the rendering in react by conditionally rendering the whole sidebar in SidebarLayout.tsx. You have the `useSmallScreen` custom hook available already and you could use that as a condition to render or not the `<aside>` tag in SidebarLayout